.github/workflows/save_atomic_files.yml

```diff
@@ -1,17 +1,26 @@
 name: save-atomic-files
 
 on: 
-  workflow_dispatch:
-  schedule:
-    - cron: '0 0 * * 0' # run at midnight every Sunday
+  push:
+    branches:
+    - '*'
+
+  pull_request:
+    branches:
+    - '*'
+  # workflow_dispatch:
+  # schedule:
+  #   - cron: '0 0 * * 0' # run at midnight every Sunday
 
 jobs:
   create-artifacts:
     uses: ./.github/workflows/bridge.yml
   
   zip_artifacts:
     needs: create-artifacts
-    if: ${{needs.create-artifacts.outputs.ref-check-trig}} == 'true'
+    if: | 
+      success() ||
+      (failure() && ${{needs.create-artifacts.outputs.ref-check-trig}} == 'true')
```

Comment:
I think you need to leave blank spaces before and after `{{ }}`, for example `${{ secrets.GITHUB_TOKEN }}`

Reply:
I see!